FeedSX/viewDataModels/postViewData/index.tsx

```diff
@@ -0,0 +1,26 @@
+export function postViewData(data: any): LMPostUI[] {
```

Comment:
Also, why input parameter is `any` , it should be the interface/class returned from data layer